src/pages/ProfilesExplorerView/components/SceneExploreServiceFlameGraph/SpanSelectorLabel.tsx

```diff
@@ -0,0 +1,30 @@
+import { css } from '@emotion/css';
+import { Button, Tooltip, useStyles2 } from '@grafana/ui';
+import React from 'react';
+
+export function SpanSelectorLabel({ spanSelector }: { spanSelector: string }) {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <div className={styles.container}>
+      <Tooltip content={`You have added a span selector to the flamegraph query (${spanSelector}).`} placement="top">
+        <span>Span selector added</span>
+      </Tooltip>
+      <Button
+        size="md"
+        fill="text"
+        variant="secondary"
+        icon="times"
+        tooltip={`Remove span selector from query`}
+        tooltipPlacement="top"
+        onClick={() => {}}
```

Comment:
Looks like we needed to use a variable for state management. But I will need to utilise your expertise here @Pitor for the best way to remove the spanSelector when this button is clicked? (not sure what way to go about this)